libraries/redshop/helper/order.php

```diff
@@ -1532,7 +1532,8 @@ public static function getOrderStatusList()
                 )
             )
             ->from($db->qn('#__redshop_order_status'))
-            ->where($db->qn('published') . ' = ' . $db->q('1'));
+            ->where($db->qn('published') . ' = ' . $db->q('1'))
+            ->order($db->qn('ordering') . ' ASC');
```

Comment:
@ducredweb : this is just a suggestion: that you can pass a param $ordering = 'ASC'. then query is:
`->order($db->qn('ordering') . ' ' . $ordering);`. In future we can use it more flexible.
Thanks.